eventsource/stores/sqlstore/store.go

```diff
@@ -27,22 +27,38 @@ func New(db *sql.DB, tableName string) eventsource.Store {
 }
 
 // Save ...
-func (store *store) Save(record eventsource.Record) (err error) {
-	return store.SaveWithContext(context.Background(), record)
+func (store *store) Save(records ...eventsource.Record) (err error) {
+	return store.SaveWithContext(context.Background(), records...)
 }
 
 // SaveWithContext ...
-func (store *store) SaveWithContext(ctx context.Context, record eventsource.Record) (err error) {
-	stmt, err := store.db.PrepareContext(ctx, fmt.Sprintf(saveSQL, store.tablename))
+func (store *store) SaveWithContext(ctx context.Context, records ...eventsource.Record) (err error) {
+	tx, err := store.db.BeginTx(ctx, nil)
 	if err != nil {
 		return
 	}
-	defer stmt.Close()
```

Comment:
I think Rollback and commit will automatically call stmt.Close for all statements in that transaction tx, see https://golang.org/src/database/sql/sql.go